keyboards/cyboard/dactyl/keymaps/singlearc_number_row/rules.mk

```diff
@@ -6,3 +6,16 @@ TAP_DANCE_ENABLE = yes
 COMBO_ENABLE = yes
 KEY_OVERRIDE_ENABLE = yes
 VIALRGB_ENABLE = yes
+
+#-----------------------------------------------------------------------------
+# sunaku
+#-----------------------------------------------------------------------------
+
+# https://docs.qmk.fm/#/feature_caps_word?id=caps-word
+CAPS_WORD_ENABLE = yes
```

Comment:
This isn't needed for the `SPLIT_USB_DETECT` fix. :policeman: This just enables [Pascal Getreuer's CapsWord feature](https://getreuer.info/posts/keyboards/caps-word/index.html).